ros1_sa_build.sh

```diff
@@ -10,24 +10,22 @@ pip3 install colcon-bundle colcon-ros-bundle
 . /opt/ros/$ROS_DISTRO/setup.sh
 
 BUILD_DIR_NAME=`basename $TRAVIS_BUILD_DIR`
-# use colcon as build tool to build the robot workspace
-cd /"$ROS_DISTRO"_ws/src/$BUILD_DIR_NAME/robot_ws/
-rosws update
-rosdep install --from-paths src --ignore-src -r -y
-colcon build --build-base build --install-base install
-colcon bundle --build-base build --install-base install --bundle-base bundle
-mv ./bundle/output.tar.gz ./bundle/robot.tar.gz
 
-# use colcon as build tool to build the simulation workspace
-cd /"$ROS_DISTRO"_ws/src/$BUILD_DIR_NAME/simulation_ws/
-rosws update
-rosdep install --from-paths src --ignore-src -r -y
-GAZEBO_POST_ROSDEP_INSTALL_SCRIPT="${SCRIPT_DIR}/gazebo/${GAZEBO_VERSION}/post_rosdep_install.sh"
-if [ -f ${GAZEBO_POST_ROSDEP_INSTALL_SCRIPT} ]; then bash ${GAZEBO_POST_ROSDEP_INSTALL_SCRIPT}; fi
-colcon build --build-base build --install-base install
-colcon bundle --build-base build --install-base install --bundle-base bundle
-mv ./bundle/output.tar.gz ./bundle/simulation.tar.gz
-
-# move the artifacts to a shared mount point
-mv /"$ROS_DISTRO"_ws/src/$BUILD_DIR_NAME/robot_ws/bundle/robot.tar.gz /shared/robot_ws.tar.gz
-mv /"$ROS_DISTRO"_ws/src/$BUILD_DIR_NAME/simulation_ws/bundle/simulation.tar.gz /shared/simulation_ws.tar.gz
+IFS=','
+for WS in $WORKSPACES
```

Comment:
Seems like this change is backwards-incompatible which means you'd have to change *every* SAs' .travis.yml before merging this, correct? Not just objecttracker / deepracer.

To make this backwards-compatible, we'd probably need to do something like '*if WORKSPACES is undefined or empty, WORKSPACES=robot_ws,simulation_ws*' (and the minor benefit is this would save one line from the .travis.yml of the common SAs).

Overall I'm fine with either approach, just need to make sure we don't break anything :)

Reply:
Good point, default added. Should be backwards compatible now